source/javascripts/controllers/awards_controller.coffee

```diff
@@ -1,2 +1,17 @@
-@EGA.controller "AwardsController", ($scope, $http) ->
-  $http.get("data/awards.json").then (res) -> $scope.awards = res.data
+EGA.controller "AwardsController", ($scope, $http) ->
+  $scope.awards = []
+  $scope.categories = []
+  $scope.categoryIsShown = {}
+  $scope.years = []
+  $scope.yearIsShown = {}
+
+  $http.get("data/awards.json").then (res) ->
+    $scope.awards = res.data
+
+    categoriesFilter = wirsing.filter['switch'].on($scope.awards, 'category')
```

Comment:
Why are you using the array accessor `['switch']` instead of just using `wirsing.filter.switch.on`…?